src/installer/image/installer.go

```diff
@@ -80,7 +128,7 @@ func (installer *Installer) InstallAgent(targetDir string) (bool, error) {
 }
 
 func (installer Installer) Cleanup() error {
```

Comment:
do we even need this anymore? (In the `interface`)

Reply:
fixed in e278a5f2